tests/checkpoint/test_checkpoint.py

```diff
@@ -561,6 +561,7 @@ def test_checkpoint_configuration_no_nesting_using_test_yaml_config(
     assert len(data_context.list_checkpoints()) == 0
 
 
+@pytest.mark.slow  # 1.74s
```

Comment:
Happy to remove these comments if we'd like - let me know!

Reply:
No these are great.
How much work would it be to make the script update the markers/comments?

Reply:
hmm not immediately sure - I'd love to automate this overall process but we unfortunately need to run the `--durations=0` call, save that output, and feed it into this script. Currently a reasonably manual process.